requirements/base.in

```diff
@@ -12,4 +12,4 @@ django-choices
 django-cors-middleware
 
 djangorestframework
-zds-schema==0.14.0
+zds-schema==0.16.1
```

Comment:
Best knap dat de input 0.16.1 is maar de output 0.16.0 ....

Reply:
Geen idee waar je 't over hebt... :wink: